lib/PrintAsObjC/PrintAsObjC.cpp

```diff
@@ -917,6 +915,62 @@ class ObjCPrinter : private DeclVisitor<ObjCPrinter>,
     }
     return hasPrintedAnything;
   }
+    
+  void printRenameForDecl(const AvailableAttr *AvAttr, const ValueDecl *D,
+                          bool includeQuotes) {
+    if (AvAttr->Rename.empty())
+      return;
+    
+    auto renamedParsedDeclName = parseDeclName(AvAttr->Rename);
+    auto renamedDeclName = renamedParsedDeclName.formDeclName(D->getASTContext());
+    
+    auto declContext = D->getDeclContext();
+    const ValueDecl *renamedFuncDecl = nullptr;
```

Comment:
This still shouldn't be called "renamedFuncDecl" when it's very often not a function. :-)

Reply:
Oh right, sorry that I missed that